src/condition.cpp

```diff
@@ -899,8 +901,7 @@ bool ConditionRegeneration::executeCondition(Creature* creature, int32_t interva
 		realHealthGain = creature->getHealth() - realHealthGain;
 
 		if (isBuff && realHealthGain > 0) {
-			Player* player = creature->getPlayer();
-			if (player) {
+			if (auto player = creature->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
			if (const auto player = creature->getPlayer()) {
```